server/config/seed.js

```diff
@@ -0,0 +1,33 @@
+import Debug from 'debug';
+import pool from '.';
+import encrypt from '../helpers/encrypt';
+
+const debug = Debug('dev');
+const { encryptPassword } = encrypt;
+
+/**
+ * Insert super admin after tables are created
+ * @name insertSuperAdmin
+ * @returns {String} details of insert
+ */
+const insertSeed = async () => {
+  const hashedPassword = encryptPassword('nonsoDrums');
```

Comment:
Haha I see what you did here! @LukasChiama

Reply:
Lol. Thanks a lot for the review. I've learned a whole lot from the team

Reply:
Hahahaha. hashing `nonsoDrums`